src/starknetCli.ts

```diff
@@ -94,17 +90,12 @@ export function runStarknetStatus(tx_hash: string, option: IOptionalNetwork & IG
   const feederGatewayUrlOption = option.feeder_gateway_url
     ? `--feeder_gateway_url ${option.feeder_gateway_url}`
     : '';
+  const command = 'starknet tx_status';
 
-  try {
-    execSync(
-      `${warpVenvPrefix} starknet tx_status --hash ${tx_hash} --network ${option.network} ${gatewayUrlOption} ${feederGatewayUrlOption}`.trim(),
-      {
-        stdio: 'inherit',
-      },
-    );
-  } catch {
-    logError('starknet tx_status failed');
-  }
+  execSyncAndLog(
+    `${warpVenvPrefix} starknet tx_status --hash ${tx_hash} --network ${option.network} ${gatewayUrlOption} ${feederGatewayUrlOption}`.trim(),
+    command,
```

Comment:
`command` is used just once. Please either use it in the line above or insert plain string in the current line